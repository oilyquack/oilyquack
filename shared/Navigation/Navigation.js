import { Link } from "gatsby";
import React from "react";

import NameTag from "./components/NameTag";
import { navigation } from "./Navigation.module.css";

const Navigation = () => (
  <header className={navigation}>
    <NameTag />
    {/* <div className={styles.navItems}>
      <Link to="/">Home</Link>
      <Link to="/blog">Blog</Link>
    </div> */}
  </header>
);

export default Navigation;
