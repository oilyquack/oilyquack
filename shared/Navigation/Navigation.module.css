.navigation {
  background: #fff;
  display: flex;
  justify-content: space-between;
  padding: 60px 5vw 60px 5vw;
  top: 0;
  width: 100%;
}

.navItems :nth-child(1) {
  margin-right: 16px;
}
