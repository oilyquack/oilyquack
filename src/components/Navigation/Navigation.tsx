import type { Component } from "solid-js";

import styles from "./Navigation.module.css";
import NameTag from "./components/NameTag";

const Navigation: Component = () => {
  return (
    <header class={styles.navigation}>
      <NameTag />
    </header>
  );
};

export default Navigation;
