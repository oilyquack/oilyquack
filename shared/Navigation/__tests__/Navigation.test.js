import { render } from "@testing-library/react";
import React from "react";

import Navigation from "../Navigation";

describe("shared : Navigation", () => {
  it("renders correctly", () => {
    const { container } = render(<Navigation />);
    expect(container).toMatchSnapshot();
  });
});
