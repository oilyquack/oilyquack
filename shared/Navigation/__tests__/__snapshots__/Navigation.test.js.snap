// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shared : Navigation renders correctly 1`] = `
<div>
  <header
    class="navigation"
  >
    <div
      class="NameTagWrapper"
      unselectable="on"
    >
      <span
        class="firstName"
      >
        <span>
          Ollie
        </span>
        <span>
          @Oily
        </span>
      </span>
      <span>
         
      </span>
      <span
        class="lastName"
      >
        <span>
          Crook
        </span>
        <span>
          Quack
        </span>
      </span>
    </div>
  </header>
</div>
`;
